hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/codec/DecodingFacade.java

```diff
@@ -61,6 +61,7 @@ public class DecodingFacade {
     private static final String UINT256_RAW_TYPE = "(uint256)";
     private static final String ADDRESS_UINT256_RAW_TYPE = "(bytes32,uint256)";
     private static final String ADDRESS_ADDRESS_UINT256_RAW_TYPE = "(bytes32,bytes32,uint256)";
+    public static final String ADDRESS_RAW_TYPE = "(bytes32)";
```

Comment:
You can use the constant types from hapi-utils - `ParsingConstants` or add there missing constants.